README.md

```diff
@@ -357,6 +359,13 @@ dependencies {
 
 ## Release notes
 
+### 3.43.0
++ Added retry in case of a RabbitMQ channel or connection error (when possible).
++ Added InterruptedException to basicConsume method signature.
++ Added additional logging for RabbitMQ errors.
++ Fixed connection recovery delay time.
++ Integration tests for these scenarios.
```

Comment:
```suggestion
### 3.42.1
+ Added retry in case of a RabbitMQ channel or connection error (when possible).
+ Added InterruptedException to basicConsume method signature.
+ Added additional logging for RabbitMQ errors.
+ Fixed connection recovery delay time.
+ Integration tests for these scenarios.
```